src/components/Users/User.jsx

```diff
@@ -0,0 +1,63 @@
+import React from "react";
+import userPhoto from "../../assets/images/user.png";
+import { NavLink } from "react-router-dom";
+import Button from "@material-ui/core/Button";
+import styled from "styled-components";
+
+const User = ({ users, followingInProgress, unfollow, follow }) => {
+  return (
+    <div>
+      {users.map((user) => (
```

Comment:
Need to use the same logic in the project. If you made a decision that you mapping your data separately - you should do it in all places. I mean like this: `const postElements = posts.map((post) =>...`